docs/graphql/manual/actions/types/index.rst

```diff
@@ -141,6 +141,11 @@ a scalar called ``Date``, you can define it like.
 These scalars can be used as arguments of the mutation or as fields of object
 types and input types.
 
+.. admonition:: Postgres scalars
+
+   The base types from Postgres can be reusable as custom scalars without explicitly declaring them.
+   For example, the ``uuid`` type from ``pgcrypto`` extension can be used to resolve a UUID data.
```

Comment:
```suggestion
   For example, the ``uuid`` type from the ``pgcrypto`` extension can be used to resolve a ``uuid`` data field.
```

Reply:
@rakeshkky I was not sure what you meant by "resolve a uuid data". I added a suggestion, please check for correctness 🙂